allennlp/training/learning_rate_schedulers.py

```diff
@@ -22,10 +23,58 @@ class LearningRateScheduler(Registrable):
     """
     This class just allows us to implement ``Registrable`` for Pytorch :class:`LRSchedulers`.
     """
+    def __init__(self, lr_scheduler) -> None:
+        self.lr_scheduler = lr_scheduler
+
     @classmethod
     def from_params(cls, optimizer: torch.optim.Optimizer, params: Params):
         scheduler = params.pop_choice("type", LearningRateScheduler.list_available())
-        return LearningRateScheduler.by_name(scheduler)(optimizer, **params.as_dict())  # type: ignore
+
+        lrs = LearningRateScheduler.by_name(scheduler)(optimizer, **params.as_dict())  # type: ignore
```

Comment:
Perhaps use `schedulers` for this variable name. In general we prefer un-abbreviated, longer variable names in the codebase.

Reply:
Cool will change now.

Reply:
@DeNeutoy I don't have 90% code coverage on the code I added. 
It is in the 80% would you be able to merge it without me meeting the 90% requirement?

Reply:
Yes no problem - the code coverage is more of a soft requirement and designed to alert us when we for instance write a test which isn't getting run, so we can spot it. This PR is certainly well enough tested.